spec/commands/add_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe "bundle add" do
+  before :each do
+    build_repo2
+
+    gemfile <<-G
+      source "file://#{gem_repo2}"
+    G
+  end
+
+  context "when version number is set" do
+    it "adds gem with provided version" do
+      bundle "add activesupport 2.3.5"
+      expect(bundled_app("Gemfile").read).to match(/gem 'activesupport', '= 2.3.5'/)
+    end
+
+    it "adds gem with provided version and version operator" do
+      update_repo2 do
+        build_gem "activesupport", "3.0"
+      end
+
+      bundle "add activesupport '> 2.3.5'"
+      expect(bundled_app("Gemfile").read).to match(/gem 'activesupport', '> 2.3.5'/)
+    end
+  end
+
+  context "when version number is not set" do
+    it "adds gem with last stable version" do
+      bundle "add activesupport"
+      expect(bundled_app("Gemfile").read).to match(/gem 'activesupport', '= 2.3.5'/)
```

Comment:
I thought we decided that the gem would be added with `~>` if no version was specified?